Tests/PointFreeTests/__Snapshots__/NavViewTests/testNav_LoggedOut.1.html

```diff
@@ -35,7 +35,17 @@
           </li>
           <li class="m-pl3 inline">
             <a href="/login"
-               class="btn medium rounded border-none pf-link-black fg-black bg-white h5 m-pl2 m-pr2">
+               class="btn
+                      medium
+                      rounded
+                      border-none
+                      text-decoration-none
+                      pf-link-black
+                      fg-black
+                      bg-white
+                      h5
+                      m-pl2
+                      m-pr2">
```

Comment:
I really like the pretty printing, but it's sooooo slow. I tried looking into ways to make it fast, but didnt get anywhere. maybe we just gotta make our own.

Reply:
Yeah the pretty-printing is crucial in verifying snapshot diffs :/